src/context/yaml/handlers/attackProtection.js

```diff
@@ -0,0 +1,16 @@
+async function parse(context) {
+  return {
+    attackProtection: context.assets.attackProtection || {}
+  };
+}
+
+async function dump(context) {
+  return {
+    attackProtection: context.assets.attackProtection || {}
+  };
+}
+
+export default {
+  parse,
+  dump
+};
```

Comment:
Is this really all that needs to be done to implement the yaml handler? A bit surprising if true 🤔 .

Reply:
Well it works 😂 , I still don't fully understand why but it does, both importing or exporting correctly.